archive/src/main/java/org/opentestsystem/rdw/archive/LocalArchiveService.java

```
@@ -98,8 +98,8 @@ public void writeResource(final String location, final InputStream is, final Pro
     @Override
     public InputStream openResource(final String location) {
         final File file = realResourceFile(location);
-        if (!file.exists()) {
-            return null;
+        if (!file.isFile()) {
+            throw new IllegalArgumentException("Invalid resource location " + location);
```

note the slight change in functionality: instead of returning null for unknown resources, it now throws IAE